docs/supportedsites.md

```diff
@@ -517,6 +517,12 @@ Consider all listed sites to potentially be NSFW.
     <td>Games</td>
     <td></td>
 </tr>
+<tr>
+    <td>Iwara</td>
+    <td>https://iwara.tv/</td>
+    <td>Profiles, videos, images, playlists, searches, tags</td>
```

Comment:
https://github.com/mikf/gallery-dl/blob/d349c248c079cc70d043a8f312e427fe4c7c03a1/docs/supportedsites.md#L3

You should run this script or `make` to generate this file's new entries.